.github/workflows/tests.yml

```diff
@@ -40,6 +40,11 @@ jobs:
         shell: bash -l {0}
         run: |
           pytest -v tests
+      - name: Test CLI
+        shell: bash -l {0}
+        run: |
+          chemprop train -i tests/data/regression/mol.csv --task-type regression --epochs 1 --num-workers 4
+          chemprop predict --checkpoint tests/data/example_model_v2.ckpt -i tests/data/regression/mol.csv --num-workers 4
```

Comment:
should be a separate PR and should be inside of `pytest`

Reply:
Thanks! I will make an issue to keep track of this.